docs/reference/scripts.mdx

```diff
@@ -4,48 +4,42 @@ Scripts let you run non-permanent Cadence scripts on the Flow blockchain. They c
 
 They always need to contain a `pub fun main()` function as an entry point to the script.
 
-`fcl.send` is a function that communicates with Access Nodes ([How to configure Access Node](../config)), it takes an array of `builder` functions, that are used to build out an [`interaction`](../../../interaction).
+`fcl.query` is a function that sends Cadence scripts to the chain and receives back decoded responses.
 
-A script is a type of interaction. We can use the `fcl.script` builder, to convert this interaction into a script.
-
-`fcl.script` is a [JavaScript Tagged Template Literal](https://styled-components.com/docs/advanced#tagged-template-literals) that we can pass Cadence code into.
+The `cadence` key inside the object sent to the `query` function is a [JavaScript Tagged Template Literal](https://styled-components.com/docs/advanced#tagged-template-literals) that we can pass Cadence code into.
 
 ### Sending your first Script
 
 In the following code snippet we are going to send a script to the Flow blockchain.
 The script is going to add two numbers, and return them.
-We can then pass the response we get back into `fcl.decode` to get back the returned value.
 
 ```javascript
 import * as fcl from "@onflow/fcl"
 
-var response = await fcl.send([
-  fcl.script`
+const response = await fcl.query({
+  cadence: `
     pub fun main(): Int {
       return 1 + 2
     }
   `
-])
+})
 
-var data = await fcl.decode(response)
-console.log(data) // 3
+console.log(response) // 3
 ```
 
 ### A more complicated Script
 
 Things like [Resources](https://docs.onflow.org/docs/cadence#resources) and [Structs](https://docs.onflow.org/docs/cadence#structures) are fairly common place in Cadence.
 
-This next example shows that `fcl.decode` out of the box knows how to convert these types of things.
-
 In the following code snippet, our script defines a struct called `Point`, it then returns a list of them.
 
 The closest thing to a Structure in JavaScript is an object. In this case when we decode this response, we would be expecting to get back an array of objects, where the objects have an `x` and `y` value.
```

Comment:
Should we still reference decoding the response since it's not exposed when using query?

Reply:
@gregsantos query decodes it for you

Reply:
Yeah, that's why i thought it might be confusing to mention in this context